builtin/mainmenu/tab_content.lua

```diff
@@ -61,14 +72,30 @@ local function get_formspec(tabview, name, tabdata)
 
 	local use_technical_names = core.settings:get_bool("show_technical_names")
 
+	local packages_with_updates = update_detector.get_all()
+	local update_icons = get_content_icons(packages_with_updates)
+	local update_count = #packages_with_updates
+	local contentdb_label
+	if update_count == 0 then
+		contentdb_label = fgettext("Browse online content")
+	elseif update_count == 1 then
+		contentdb_label = fgettext("Browse online content [1 update]")
+	else
+		contentdb_label = fgettext("Browse online content [$1 updates]", update_count)
```

Comment:
I'd still prefer reducing this to `"Browse online content [$1]"`. `"Browse online content [$1 updates]"` is too much text on Android (doesn't fit on the button, especially when translated).